src/Parser.js

```diff
@@ -78,22 +80,28 @@ export function parse (value, points, declarations, rules, rulesets) {
 
 				index = length = offset = 0, ampersand = 1, type = temporary = ''
```

Comment:
the `variable = 1` should go here i.e `variable = ampersand = 1`

Reply:
you are most likely right, can you think of any nice test for this?

Reply:
It's not a semantic change, we just reset everything on one place.